d2bs/kolbot/libs/bots/DiabloHelper.js

```diff
@@ -457,23 +546,22 @@ CSLoop:
 
 	Pather.moveTo(7774, 5305);
 	Attack.clear(35, 0, false, this.sort);
-	this.vizierSeal();
-	this.seisSeal();
-	Precast.doPrecast(true);
-	this.infectorSeal();
-
-	switch (me.classid) {
-	case 1:
-		Pather.moveTo(7793, 5291);
-
-		break;
-	default:
-		Pather.moveTo(7788, 5292);
-
-		break;
+	clearSeals(Config.DiabloHelper.SealOrder);
+
+	try {
+		print("Attempting to find Diablo");
+		this.diabloPrep();
+	} catch (error) {
```

Comment:
Same as before, cant we do a corpse check? =)

Reply:
I wasn't aware there was a corpse check :) I'll look into that.

The reason for the try/catch though was because I noticed a situation where my sorc would skip over Seis because of the cold immune, and then if no other char kills Seis, they all go wait for Diablo to come but since all the seals weren't cleared, Diablo wouldn't awaken and they'll all error out and move to the next script.

Another scenario where this try/catch comes in handy, specific to `SealOrder` is if a character that was assigned to a certain seal drops, and no one else clears it, they'll double-back and clear it if Diablo doesn't show up.